### PR TITLE
Debugging Node: Assignment resources order changed

### LINKED ISSUE #25001: NodeJS: Debugging Node assignment resources order suggestion
### Describe your suggestion

Hi, I am currently going through the Debugging Node lesson, and when I got to the assignment section I started with the first link which links to the official VS Code Node debugger documentation. The problm is that I couldn't keep up with the content there. I decided to start with the 3rd resource which is a video to debugging in VS Code, and came back to the documentation and it was much easier. So I suggest putting the video just above the Documentation link, that will make understanding the concepts much easier.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/nodejs-debugging-node

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### (Optional) Discord Name

Chrollo#7144

### (Optional) Additional Comments

Thank u so much for this Curriculum <3

### PR DESCRIPTION
**Pull request by roy9495**
## Because

In the Debugging Node lesson, the video for debugging in VS Code is more helpful before going through the documentation.

## This PR

-Puts the video for debugging in VS Code section before the [VS Code Node debugger documentation](https://code.visualstudio.com/docs/nodejs/nodejs-debugging)

## Issue

Closes #25001 

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: nodeJS/introduction_to_nodeJS/debugging_node.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ By the end of this lesson, you should be able to do the following:
 
 <div class="lesson-content__panel" markdown="1">
 
+- Watch [this video tutorial](https://www.youtube.com/watch?v=2oFKNL7vYV8&ab_channel=VisualStudioCode) to see the process in action.
```
Comment (ManonLef) [inline nodeJS/introduction_to_nodeJS/debugging_node.md:15]:
```suggestion
- Watch [this video tutorial](https://www.youtube.com/watch?v=2oFKNL7vYV8&ab_channel=VisualStudioCode) to see the Node.js debugging in VS Code process in action.
```

Looks good to me and thanks for taking this on. However, I think making this a bit more descriptive might be helpful. What do you think @roy9495 ?

Comment (roy9495) [reply nodeJS/introduction_to_nodeJS/debugging_node.md:15]: Yes, This description will be more specific. I will commit this.

Comment (ManonLef) [reply nodeJS/introduction_to_nodeJS/debugging_node.md:15]: You can either click the `commit suggestion` button below my review, or make a commit yourself and push it :)

Comment (roy9495) [reply nodeJS/introduction_to_nodeJS/debugging_node.md]: Sorry for the delay .

Comment (ManonLef) [reply nodeJS/introduction_to_nodeJS/debugging_node.md]: no need to apologise at all! Thanks for doing this :)

**Pull request merged**